app/questionnaire/placeholder_renderer.py

```diff
@@ -99,7 +104,10 @@ def render_placeholder(
         return formatted_placeholder_data
 
     def render(
-        self, dict_to_render: Mapping[str, Any], list_item_id: Optional[str]
+        self,
+        dict_to_render: Mapping[str, Any],
```

Comment:
```suggestion
        self,
        *,
        dict_to_render: Mapping[str, Any],
```
Enforce named attributes?

Reply:
Enforcing now and updated in relevant calls 👍